yaml/pois.yaml

```diff
@@ -562,14 +562,34 @@ filters:
       tier: 3
   # university
   - filter: {amenity: university}
-    min_zoom: { min: [ { max: [ { sum: [ { col: zoom }, 2.55 ] }, *tier3_min_zoom ] }, 16 ] }
+    min_zoom:
+      lookup:
+        key: { col: way_area }
+        op: '>='
+        table:
+          - [ 10, 10000000 ]
+          - [ 11,  5000000 ]
+          - [ 12,  1000000 ]
+          - [ 13,   500000 ]
+          - [ 14,   500000 ]
+          - [ 15,   100000 ]
+        default: 16
     output:
       <<: *output_properties
-      kind: university
+      kind: { col: amenity }
```

Comment:
This is a no-op, and the old way is more legible.

Reply:
Ooops, yeah. Thanks for spotting that. I had tried to merge the `university` and `college` parts, but it turned out to not be possible with the area thresholds we wanted and I must have forgotten to put that back the way it was.